python/src/MeshDomain_doc.i.in

```diff
@@ -0,0 +1,50 @@
+%feature("docstring") OT::MeshDomain
+"Adaptor to convert a Mesh and a Domain.
```

Comment:
... a Mesh to a Domain

Reply:
Thanks, since a rebase is needed, I will put this change directly into the first commit.